ui/components/multichain/nft-item/nft-item.test.js

```diff
@@ -1,70 +1,49 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import { fireEvent } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
+import mockState from '../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
 import { NftItem } from '.';
 
 describe('NftItem component', () => {
-  const mockOnClick = jest.fn();
-
-  afterEach(() => {
-    jest.clearAllMocks();
-  });
-
-  it('renders correctly with an image source', () => {
-    const { getByTestId } = render(
-      <NftItem
-        alt="Test Alt"
-        backgroundColor="red"
-        name="Test NFT"
-        src="test-src"
-        networkName="Test Network"
-        networkSrc="test-network-src"
-        tokenId="1"
-        onClick={mockOnClick}
-      />,
-    );
-
-    expect(getByTestId('nft-item')).toBeInTheDocument();
-    expect(getByTestId('nft-network-badge')).toBeInTheDocument();
-    expect(getByTestId('nft-image')).toBeInTheDocument();
-    expect(getByTestId('nft-image')).toHaveAttribute('src', 'test-src');
-  });
-
-  it('renders correctly with default image when no image source is provided', () => {
-    const { getByTestId, queryByTestId } = render(
-      <NftItem
-        alt="Test Alt"
-        backgroundColor="red"
-        name="Test NFT"
-        src=""
-        networkName="Test Network"
-        networkSrc="test-network-src"
-        tokenId="1"
-        onClick={mockOnClick}
-      />,
-    );
-
-    expect(getByTestId('nft-item')).toBeInTheDocument();
-    expect(getByTestId('nft-network-badge')).toBeInTheDocument();
-    expect(queryByTestId('nft-image')).not.toBeInTheDocument();
-    expect(getByTestId('nft-default-image')).toBeInTheDocument();
-  });
-
-  it('calls onClick when the NFT image is clicked', () => {
-    const { getByTestId } = render(
-      <NftItem
-        alt="Test Alt"
-        backgroundColor="red"
-        name="Test NFT"
-        src="test-src"
-        networkName="Test Network"
-        networkSrc="test-network-src"
-        tokenId="1"
-        onClick={mockOnClick}
-      />,
-    );
-
-    fireEvent.click(getByTestId('nft-image'));
-    expect(mockOnClick).toHaveBeenCalled();
+  const store = configureStore()(mockState);
+  describe('render', () => {
+    const props = {
```

Comment:
Since we are using a selector to get the color, we need to use store here to get the tests passing. 

I added an extra change to use the same `props ` and `renderWithProvider` here rather than passing it again for each test just to improve code quality.